src/quicklist.h

```diff
@@ -107,7 +107,7 @@ typedef struct quicklist {
     quicklistNode *tail;
     unsigned long count;        /* total count of all entries in all listpacks */
     unsigned long len;          /* number of quicklistNodes */
-    int fill : QL_FILL_BITS;              /* fill factor for individual nodes */
+    unsigned int fill : QL_FILL_BITS;     /* fill factor for individual nodes */
```

Comment:
https://github.com/enjoy-binbin/redis/runs/4578156546?check_suite_focus=true#step:10:2021
look like this will break the `./src/redis-server test quicklist --accurate`